httpbp/prometheus.go

```diff
@@ -16,6 +16,22 @@ const (
 	endpointLabel   = "http_endpoint"
 )
 
+var ResponseSizeBuckets = []float64{
```

Comment:
```suggestion
// should be kept in sync with the buckets defined in https://github.snooguts.net/reddit-go/httpbp/blob/main/httpbp/internal/httpmetrics/metrics.go
var ResponseSizeBuckets = []float64{
```

Reply:
🔕 I think Yuxuan had also suggested not exporting the buckets 🤷

Reply:
Updated!